packages/recurring-payments-job/src/recurring-payments-processor.js

```diff
@@ -1,18 +1,28 @@
 import moment from 'moment-timezone'
 import { SERVICE_LOCAL_TIME } from '@defra-fish/business-rules-lib'
 import { salesApi } from '@defra-fish/connectors-lib'
-import { sendPayment } from './services/govuk-pay-service.js'
+import { getPaymentStatus, sendPayment } from './services/govuk-pay-service.js'
 
+const PAYMENT_STATUS_DELAY = 60000
 const payments = []
 
+const timeout = ms => {
+  return new Promise(resolve => {
+    setTimeout(resolve, ms)
+  })
+}
+
 export const processRecurringPayments = async () => {
   if (process.env.RUN_RECURRING_PAYMENTS?.toLowerCase() === 'true') {
     console.log('Recurring Payments job enabled')
     const date = new Date().toISOString().split('T')[0]
     const response = await salesApi.getDueRecurringPayments(date)
     console.log('Recurring Payments found: ', response)
     await Promise.all(response.map(record => processRecurringPayment(record)))
-    console.log('Recurring Payments processed:', payments)
+    if (response.length > 0) {
+      timeout(PAYMENT_STATUS_DELAY)
```

Comment:
you're setting the timeout, but not awaiting it, so the next line will be called immediately. There's also no test for the delay.